term_proposal/chengscott/proposal.md

```diff
@@ -0,0 +1,35 @@
+# nsd PageRank
+
+## Background
+
+PageRank was an algorithm used by Google to rank the importance of resulting web pages from their search engine. PageRank simulates Web surfers, starting at a random page and randomly browsing around, would tend to converge and aggregated, and this process is allowed to iterate many times.
+
+The algorithm accepts a directed graph as the input, where vertices and edges represents web pages and links between, respectively; It outputs PageRank scores for each vertices, which represents the likelihood of a page being visited by random web suffering. The problem can be reformulated in matrix calculations:
```

Comment:
Did you mean to say suffering or surfing?

Reply:
Thank you. surfing, it's a typo.